pkg/kapp/clusterapply/change_set_view.go

```diff
@@ -48,3 +53,55 @@ func (v *ChangeSetView) Print(ui ui.UI) {
 func (v *ChangeSetView) Summary() string {
 	return v.changesView.Summary() // assumes Print was used before
 }
+
+func (v ChangeSetView) PrintCompleteYamlToBeApplied(ui ui.UI, conf ctlconf.Conf) error {
+	for _, view := range v.changeViews {
+		op := view.ApplyOp()
+		resYAML := ""
+		opAndResDesc := fmt.Sprintf("# %s: %s", view.ApplyOp(), view.Resource().Description())
+		strategy, err := view.ApplyStrategyOp()
+		if err != nil {
+			return err
+		}
+
+		switch op {
+		case ClusterChangeApplyOpNoop:
+			continue
+		case ClusterChangeApplyOpDelete:
+			if strategy == deleteStrategyPlainAnnValue {
+				opAndResDesc = color.RedString("%s", opAndResDesc)
+			} else {
+				opAndResDesc = color.RedString("%s (strategy: %s)", opAndResDesc, strategy)
+			}
+		case ClusterChangeApplyOpExists:
+			opAndResDesc = color.GreenString("%s", opAndResDesc)
+		default:
+			if strategy != "" {
```

Comment:
We can use `createStrategyPlainAnnValue` and `updateStrategyPlainAnnValue`, similar to `deleteStrategyPlainAnnValue`.

Reply:
updated